tests/integration/Core/Framework/DataAbstractionLayer/Field/EnumerationField/TestIntegerEnumeration.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Shopware\Tests\Integration\Core\Framework\DataAbstractionLayer\Field\EnumerationField;
```

Comment:
```suggestion
namespace Shopware\Tests\Integration\Core\Framework\DataAbstractionLayer\Field\EnumField;
```

also change the directory name accordingly :wink: